src/utilities/_hypre_utilities.h

```diff
@@ -2880,3 +2880,4 @@ hypre_UnorderedBigIntMapPutIfAbsent( hypre_UnorderedBigIntMap *m,
 #endif
 
 #endif
+
```

Comment:
```suggestion
```

Reply:
@prj- We don't directly edit this file which is generated by a script. Is there a problem for having an extra newline? Thanks!